spec/app/lib/bing_api_v7_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe BingApiV7 do
+  describe '#search', vcr: true do
+    context 'with valid response data' do
+      let!(:company) { Company.create(domain: 'softserve') }
+      let!(:bing) { BingApiV7.new(company) }
+      subject { bing.search }
```

Comment:
you'd also test for pages creation and validity since your class actually creates pages for company

Reply:
so, if i have to bcs of no way, or it should be done in another way instead?

Reply:
other way - do not user subject
call "bing.search" in "it" method, it will be more clear

Reply:
why there is no way? subject gets evaluated only when you call `subject` within test example..